packages/repository/src/repositories/legacy-juggler-bridge.ts

```diff
@@ -528,6 +536,56 @@ export class DefaultCrudRepository<
     return includeRelatedModels<T, Relations>(this, entities, include, options);
   }
 
+  protected async entityToData<R extends T>(
```

Comment:
I like this new API 👏 

Can you please add a tsdoc comment for our users?

Reply:
added :)